src/weather.h

```diff
@@ -171,7 +114,14 @@ bool dangerous( weather_type type );
 precip_class precip( weather_type type );
 bool rains( weather_type type );
 bool acidic( weather_type type );
-weather_effect_fn effect( weather_type type );
+std::vector<std::pair<std::string, int>> effects( weather_type type );
+std::string tiles_animation( weather_type type );
+weather_animation_t get_weather_animation( weather_type type );
```

Comment:
```suggestion
weather_animation_t get_animation( weather_type type );
```
This is called as `weather::get_weather_animation`. The second "weather" is redundant.